### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int):
```
Comment (Andriy-Makarenko) [inline app/main.py:4]: Best practice is to add type annotation for constructor as well, not only for arguments
-> None

**Review by Andriy-Makarenko — CHANGES_REQUESTED** [review]

**Comment by HStakhiv** [comment]
Done!

**Review by Andriy-Makarenko — APPROVED** [review]
Good job

**Review by Y-Havryliv — APPROVED** [review]
GJ!

**Pull request closed without merging**